package.json

```diff
@@ -75,11 +74,11 @@
     "style-loader": "^2.0.0",
     "stylelint": "^13.6.1",
     "stylelint-config-standard": "^20.0.0",
-    "terser-webpack-plugin": "^4.0.0",
-    "webpack": "^4.44.0",
+    "terser-webpack-plugin": "^5.0.3",
+    "webpack": "^5.8.0",
```

Comment:
I didn't realize that they were already on 5.8.  For some reason I thought that v5 was a recent thing.